src/components/editor/Editor.jsx

```diff
@@ -61,7 +61,9 @@ class Editor extends Component {
   }
 
   // The ld4p group is only for templates
-  groupsToSaveInto = () => Config.groupsInSinopia.filter(group => group[0] !== 'ld4p')
+  groupsToSaveInto = () => Object.entries(Config.groupsInSinopia)
+    .filter(([groupSlug]) => groupSlug !== 'ld4p')
```

Comment:
i thought the object destructuring on the array var made things a bit more readable, since it gave a var name for the thing being compared, instead of just an array subscript